event_routing_backends/processors/mixins/base_transformer.py

```diff
@@ -159,13 +160,16 @@ def get_data(self, key, required=False):
                     }
                 }
         """
-        result = get_value_from_dotted_path(self.event, key)
+        if '.' in key:
+            result = get_value_from_dotted_path(self.event, key)
+        else:
+            result = self.find_nested(self.event, key)
         if result is None:
             if not required:
-                logger.info('Could not get data for %s in event "%s"', key, self.get_data('name', True))
+                logger.info('Could not get data for %s in event "%s"', key, self.event.get('name', None))
```

Comment:
why we are using `self.event` instead of `self.get_data` here and below?.

Reply:
As we have modified implementation. There is a chance of name key being on inner level at video{name:'xyz'}. Other than that there is a chance that name doesn't exist in the event in that case it will stuck in infinite loop